epilogue-processor/src/test/java/edu/wpi/first/epilogue/processor/AnnotationProcessorTest.java

```diff
@@ -1163,7 +1163,9 @@ private void assertLoggerGenerates(String loggedClassContent, String loggerClass
             .orElseThrow(() -> new IllegalStateException("Logger file was not generated!"));
     try {
       var content = generatedFile.getCharContent(false);
-      assertEquals(loggerClassContent, content);
+      assertEquals(
+          loggerClassContent.replace("\r", ""),
```

Comment:
I usually choose to replace "\r\n" with "\n". You can pretty much always guarantee \r\n is a new line, but its _possible_ \r by itself is expected (even though unlikely)

Reply:
The google test compiler won't emit carriage returns, so this should be fine

Reply:
I've changed it, just to be safe

Reply:
The new update won't work. Because the constant string being compared to will still be `\n`, but now the input string won't have any line endings at all. Replace `\r\n` with `\n`, not with nothing.